Makefile

```diff
@@ -46,6 +46,9 @@ bdist_wheel:
 	${RUN_COMMAND} ./setup.py bdist_wheel
 	# ${RUN_COMMAND} auditwheel repair dist/*.whl
 
+build_deb:
+	${RUN_COMMAND} gbp buildpackage -us -uc
```

Comment:
Can you please make sure that the .deb package lands under `dist/`? It makes it then easy to deploy the package with all other packages.

Reply:
It lands in the directory above the git source code. I am not aware of any option to tell debbuild to output in a selected directory. I'll check.

Reply:
I was actually simply thinking of copying/moving the generated file to the right place.

Reply:
OK, this is now done in travis. This should be good enough to get merged now. The makefile and travis can be tweaked more later.